content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,278 @@
+## Overview
+Datadog Synthetics offers two different ways to monitor your applications: API tests to monitor the uptime of your API endpoints and browser tests to check key user journeys. Your tests can be run either from managed locations or from private locations. Synthetics helps you ensure uptime, identify regional issues, and make sure key web transactions can be performed on your application.
+
+{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" responsive="true">}}
+
+By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing, as experienced by your users. The [Synthetics][1] homepage details all of this information to give you real-time status updates, response times, and uptimes.
+
+This guide shows you how to set up your first Synthetics test with Datadog. Follow the sections below to configure an API test, browser test, or private location for internal-facing apps or private URLs.
+
+## Prerequisites
+
+Browser tests can be only recorded from **[Google Chrome][2]**. Datadog will prompt you to download the Datadog Record Test extension during the creation of the test. You can also [download this extension directly from the Chrome Web Store][3].
+
+## Create a Datadog account
+
+If you haven't already, create a [Datadog account][4].
+
+## Configure your first test
+
+### Create an API test
```

Comment:
I’m sorry I’m going to be annoying :sweat_smile: 
But regarding the general structure of the getting started, I would have expected something less comprehensive and more scoped to one or two specific examples. For instance on the logs’ one https://docs.datadoghq.com/getting_started/logs/?tab=ussite they only talk about manually sending logs & sending logs from the Agent, and don't spend too much on going into each specific parameter that can be set. So I was expecting something a bit similar?
Instead of going through all the different options of a test, I think maybe we should just stick to one of the API tests, for instance HTTP and just go through the creation of it without detailing all the options. I made a draft to show you what I was thinking of: https://docs.google.com/document/d/1LTmJXw7vXSSC8JUzkHP37YVcSVjMrUCLahNcTH4NAdk/edit?usp=sharing
Can you have a look and let me know what you think? If you feel like this does not make sense from a doc perspective, let's discuss that as I'm curious what are the best practices for this kind of material!
If you feel like this makes sense, it'd be cool to have something similar for the browser test & private location sections as well.